uint8 request_id
uint8 event_id
---
bool is_accepted
ConfirmEventRequest confirmed_event_request
RejectEventRequest rejected_event_request